doc/diagrams/c4-model/model.dsl

```diff
@@ -51,5 +51,25 @@ measurementsDomain = group "Measurements Domain" {
 }
 
 transferAgreementsDomain = group "Transfer Agreements Domain" {
+    taApi = container "Transfer Agreement API" "" ".NET Web Api" {
```

Comment:
There is a component missing for the background job that removes old invitations